SkietbaanBE/SkietbaanBE/Controllers/UserController.cs

```diff
@@ -60,46 +60,35 @@ public async Task<IActionResult> AddUser([FromBody] User user)
                 return new BadRequestObjectResult("user cannot be null");
             }
         }
-        // PUT: api/User/
+        // PUT: api/User
         [HttpPut("{id}")]
         public async Task<IActionResult> UpdateUser(int id, [FromBody] User user)
```

Comment:
Actually use the passed in id parameter otherwise why are you passing it in

Reply:
i changed it